hat/assets/js/apps/Iaso/domains/users/components/Filters.js

```diff
@@ -33,7 +33,7 @@ const Filters = ({ baseUrl, params }) => {
     const [initialOrgUnitId, setInitialOrgUnitId] = useState(params?.location);
     const { data: dropdown, isFetching } = useGetPermissionsDropDown();
     const { data: initialOrgUnit } = useGetOrgUnit(initialOrgUnitId);
-    const { data: orgUnitTypeDropdown, isFetchingOuType } =
+    const { data: orgUnitTypeDropdown, isFetchingOuTypes } =
```

Comment:
Hook`useGetOrgUnitTypes` result is an object like this `{data:[...], isFetching: true}` you can rename it and use it like this: `{data: orgUnitTypeDropdown, isFetching: isFetchingOuTypes}` .
In your case `isFetchingOuTypes`will always be `undefined` 😄